scripts/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -33,9 +33,10 @@ CAPI_REL_TO_VERSION="$(get_latest_capi_release)" || true
 # CAPI release version which we upgrade to.
 export CAPI_REL_TO_VERSION
 
-export FROM_K8S_VERSION="v1.21.1"
+export FROM_K8S_VERSION="v1.21.2"
 export KUBERNETES_VERSION=${FROM_K8S_VERSION}
-export UPGRADED_K8S_VERSION="v1.21.2"
+export UPGRADED_K8S_VERSION="v1.21.3"
```

Comment:
this has a conflict with #767. Basically two different versions are set for `UPGRADED_K8S_VERSION` depending which one will go in first, the second needs rebase. Do we keep this as is in this PR?

Reply:
@furkatgofurov7 we can let #767 to go in first. I will rebase and test after that!